docs/package.json

```
@@ -15,7 +15,8 @@
         "build:sass": "sass --style=compressed src/assets/scss:src/assets/css --no-source-map",
         "build:eleventy": "npx @11ty/eleventy",
         "start": "npm-run-all build:sass --parallel watch:*",
-        "build": "npm-run-all build:sass build:eleventy images"
+        "build": "npm-run-all build:sass build:eleventy images",
+        "test": "stylelint \"**/*.scss\""
```

```suggestion
        "lint:scss": "stylelint \"**/*.scss\""
```

We are trying to unify the script naming across repos. I feel like it'd be better if we go with `lint:scss`
https://github.com/eslint/eslint.org/blob/dae022f4ddd4a31b4fe5c9bb90e650205ab128c4/package.json#L32